docs/docs/guides/merge-multiple-db-secrets.mdx

```diff
@@ -0,0 +1,64 @@
+---
+id: merge-multiple-db-secrets
+title: Merge multiple Hydra instances with different system.secrets
+sidebar_label: Merge multiple system.secrets
+---
+
+This guide provides practical steps to merge multiple Ory Hydra Postgres
+database instances with different `system.secret` values into one instance with
+one `system.secret`.
+
+The `system.secret` or `$SECRETS_SYSTEM` is an environmental variable, that is
+used to encrypt Ory Hydras database.
+
+If you are looking for information on how to change (rotate) the
+`system.secret`, please refer to the
+[Secrets and Key Rotation Guide](https://www.ory.sh/hydra/docs/guides/secrets-key-rotation/#rotation-of-hmac-token-signing-and-database-and-cookie-encryption-keys).
+
+First we take all the system.secret keys and add them to the target instance
+environment variables like so:
+
+`SECRETS_SYSTEM=new-secret,secret-1,secret-2,secret-3,secret-n,secret-n+1`
+
+Then we run this
+[pg_dump command](https://www.postgresql.org/docs/9.3/app-pgdump.html) against
+the databases we need to migrate:
+
+```
+pg_dump --verbose -a  \
+	--format=p \
+	--no-owner \
+	--no-acl \
+	--quote-all-identifiers \
+	-t hydra_client \
+	-t hydra_oauth2_authentication_session \
+	-t hydra_oauth2_authentication_request \
+	-t hydra_oauth2_authentication_request_handled \
+	-t hydra_oauth2_consent_request \
+	-t hydra_oauth2_consent_request_handled \
+	-t hydra_oauth2_code \
+	-t hydra_oauth2_access \
+	-t hydra_oauth2_refresh \
+	-f hydra-dump.sql \
+	<DSN>
+```
+
+Then we open each of the dump files and manually edit `hydra_clients.pk` be
+entirely set to the value `DEFAULT`
+([SQL keyword](https://www.w3schools.com/sql/sql_ref_default.asp)) to make it
+follow the sequence in the target DB correctly.
+
+We also delete the last line of the dump, since it sets the value of the primary
+key sequence to the number of the donor database. So we need to delete this
+line.
```

Comment:
If there's any way of putting emphasis on this that would be good, it's very important.
This whole manual edits section is very important, but the first part will most likely just fail importing and not cause any harm if not done correctly, and can be recovered with a correct import afterwards (with a few harmless duplicate key errors). Where as if you forget to remove this line it will reset the sequence of `hydra_clients.pk` to whatever was in the source DB potentially breaking client creation going forwards until the sequence gets back in sync

Reply:
Thanks for the input! 
I put in some warnings, to make clear that this can potentially break client creation if you mess it up.